graphql/resolvers.go

```diff
@@ -1979,16 +1979,21 @@ func (r *mutationResolver) RestartPatch(ctx context.Context, patchID string, abo
 	return &patchID, nil
 }
 
-func (r *mutationResolver) SetPatchPriority(ctx context.Context, patchID string, priority int) (*string, error) {
+func (r *mutationResolver) SetPatchPriority(ctx context.Context, patchIds []string, priority int) ([]string, error) {
```

Comment:
I know this pr isn't complete yet but this would result in a non backwards compatible api change.  I think this is a good opportunity to rename this resolver to `SetVersionPriority` since that is a more accurate description of what it does.

Reply:
In order to keep this backwards compatible i would keep this resolver as is and create a second resolver called `SetVersionPriority` that accepts the updated parameters. In a follow up PR we can remove the `SetPatchPriority` resolver when the front end changes have been deployed.